### PR TITLE
KFSPTS-30684: Reinstate base code functionality that sends failure and summary system email notifications when payments are formatted in PDP.

### PR DESCRIPTION
**Pull request by nkimble**
This PR requires nonprod-sql PR #1332. 
Please do not merge this code until both are ready to be merged.

### CODE REVIEW

File: src/main/java/com/rsmart/kuali/kfs/pdp/service/impl/AchBundlerFormatServiceImpl.java [hunk 1]
```diff
@@ -90,7 +87,8 @@ public AchBundlerFormatServiceImpl() {
      */
     @Override
     public void performFormat(Integer processId) throws FormatException {
-        LOG.debug("performFormat() started - ACH Bundler Mod");
+        LOG.info("performFormat() started - ACH Bundler Mod for processId {}", processId);
+        final String pdpFormatFailureToEmailAddress = this.getAchBundlerHelperService().getPdpFormatFailureToEmailAddress();
```
Comment (db483) [inline src/main/java/com/rsmart/kuali/kfs/pdp/service/impl/AchBundlerFormatServiceImpl.java:91]: Should 'this.' be removed? I think it is unnecessary and KualiCo cleaned these up in base code.

**Review by db483 — CHANGES_REQUESTED** [review]
Changes look good. I just had one small suggestion for improvement. Thank you.

File: src/main/java/com/rsmart/kuali/kfs/pdp/service/impl/AchBundlerFormatServiceImpl.java (hunk 1 above)
Comment (nkimble) [reply src/main/java/com/rsmart/kuali/kfs/pdp/service/impl/AchBundlerFormatServiceImpl.java:91]: Removing

**Review by db483 — APPROVED** [review]
Changes look good and unit tests pass.

**Pull request merged**